src/poster/poster.scss

```diff
@@ -70,8 +70,7 @@
     width: 100%;
     overflow: hidden;
     background-color: var(--bg-primary);
-    background-size: cover;
-    background-position: center;
+    object-fit: cover;
```

Comment:
Not sure about the styling here.. 
Is it necessary to hide overflow, and have a background?
Could cover not lead to cases where parts of the logo could be cut off?
I guess we should seek a solution here, were we show the full image.